root/static/scripts/tests/Control/URLCleanup.js

```diff
@@ -4892,6 +4892,14 @@ limited_link_type_combinations: [
              input_entity_type: 'artist',
     expected_relationship_type: 'otherdatabases',
             expected_clean_url: 'https://www.abc.net.au/triplejunearthed/artist/sophisticated-dingo/',
+},
```

Comment:
Super minor, but this closing bracket (and the next) are wrongly indented :)

Reply:
Good catch! This is an easy fix, so I can definitely adjust this.

Reply:
This is done.